verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -120,8 +144,9 @@ export default class CasesController {
             }
             return false;
         } else {
-            // Remove any query as we shouldn't store it.
+            // Remove any left over field as the schema doesn't allow unknown fields.
```

Comment:
For now, is this strictly necessary? Seems like something we'd added because we got an error, but with Mongoose `strict=true` (rather than `throw`), I thought we just silently dropped these.